types/zxcvbn/index.d.ts

```diff
@@ -0,0 +1,209 @@
+// Type definitions for zxcvbn 4.4
+// Project: https://github.com/dropbox/zxcvbn#readme
+// Definitions by: Matt Traynham <https://github.com/mtraynham>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace zxcvbn;
+
+export = zxcvbn;
+
+declare function zxcvbn(password: string, userInputs?: string[]): zxcvbn.ZXCVBNResult;
+
+declare namespace zxcvbn {
+    interface ZXCVBNResult {
+        /**
+         * estimated guesses needed to crack password
+         */
+        guesses: number;
+
+        /**
+         * order of magnitude of result.guesses
+         */
+        guesses_log10: number;
+
+        /**
+         * dictionary of back-of-the-envelope crack time
+         * estimations, in seconds, based on a few scenarios.
+         */
+        crack_times_seconds: ZXCVBNAttackTime;
+
+        /**
+         * same keys as result.crack_times_seconds,
+         * with friendlier display string values:
+         * "less than a second", "3 hours", "centuries", etc.
+         */
+        crack_times_display: ZXCVBNAttackTime;
+
+        /**
+         * Integer from 0-4 (useful for implementing a strength bar):
+         * 0 too guessable: risky password. (guesses < 10^3)
+         * 1 very guessable: protection from throttled online attacks. (guesses < 10^6)
+         * 2 somewhat guessable: protection from unthrottled online attacks. (guesses < 10^8)
+         * 3 safely unguessable: moderate protection from offline slow-hash scenario. (guesses < 10^10)
+         * 4 very unguessable: strong protection from offline slow-hash scenario. (guesses >= 10^10)
+         */
+        score: number;
```

Comment:
Convert this to enum and use that instead of setting the type of score as number?

Reply:
`score: 0 | 1 | 2 | 3 | 4;`, sound good to you?

Reply:
Type is fine.